pkg/genericapiserver/server/serve.go

```diff
@@ -93,13 +93,13 @@ func (s *GenericAPIServer) serveInsecurely(stopCh <-chan struct{}) error {
 	}
 	glog.Infof("Serving insecurely on %s", s.InsecureServingInfo.BindAddress)
 	var err error
-	s.effectiveInsecurePort, err = runServer(insecureServer, s.InsecureServingInfo.BindNetwork, stopCh)
+	s.effectiveInsecurePort, err = runServer(insecureServer, s.InsecureServingInfo.BindNetwork, &s.InsecureServingInfo.ipLimit, stopCh)
```

Comment:
It is effectively not being turned on for the insecure port. The InsecureServingInfo is never really initialized (its all defaults). The default behavior is wide open with no limits applied.